jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -463,7 +464,7 @@ private Trigger createTrigger(Server server) throws ParseException {
 			trigger = TriggerUtils.makeSecondlyTrigger(runPeriod);
 			// TODO replace Quartz with a ScheduledExecutorService
 		}
-		trigger.setName(server.getHost() + ":" + server.getPort() + "-" + Long.toString(System.nanoTime()));
+		trigger.setName(server.getHost() + ":" + server.getPort() + "-" + Long.toString(System.nanoTime()) + "-" + UUID.randomUUID().toString().substring(0, 8));
```

Comment:
To generate a short  random String, something like `Long.toHexString(random.nextLong())` may be simpler et quicker

Reply:
Thanks @gquintana should be changed now.